components/scream/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -44,7 +44,7 @@ namespace scream {
          * objects. So we will initialize once, and then just update values
          * at runtime.
          */
-        GasConcs gas_concs;
+        //GasConcs gas_concs;
```

Comment:
Should this be removed then?

Reply:
I'm leaning towards leaving this in. I'm still not 100% sure how to handle this and I think with Aaron's new initializer PR this might make more sense. I think it might be best to revisit after that PR goes in.

Reply:
I'm leaning towards keeping it for now. I'm actually not 100% sure how I want to handle this, and I suspect it may change  with the new initializer, once that PR goes in. So will revisit when we revisit initialization of radiation fields.